policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/decoupled.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   reference:
     - title: Indiana General Assembly Laws
       href: https://iga.in.gov/laws/2021/ic/titles/6#6-3.1-21
+    - title: 2023 IT-40 Tax Form Instructions Booklet, pg.3
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
+    - title: 2023 IT-40 Income Tax Form, Schedule IN-EIC, pg. 2
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
+
 values:
   2010-01-01: false
   2011-01-01: true
+  2023-01-01: false
```

Comment:
add a 2023 test to the variable that is affected

Reply:
I still can't run the test due to the problem I had a couple of weeks before; I did check the test section and it's this screenshot
<img width="487" alt="Screenshot 2024-07-21 at 3 44 25 PM" src="https://github.com/user-attachments/assets/c883a1ed-293f-4df1-bfd5-5e451acb62be">
I think we need to modify the outdated parameters (0.09 and it should be 0.10 now)